psutil/_psutil_posix.c

```diff
@@ -16,7 +16,7 @@
 #include <net/if.h>
 
 #ifdef PSUTIL_SUNOS10
-#include "arch/solaris10/ifaddrs.h"
+#include "arch/solaris/v10/ifaddrs.h"
```

Comment:
Doesn't this break code for non-sparc architectures? Have you tested this on something other than sparc?